src/native/minipal/cpufeatures.h

```diff
@@ -62,6 +62,15 @@ static_assert((1 << ARM64_ATOMICS_FEATURE_FLAG_BIT) == ARM64IntrinsicConstants_A
 
 #endif // HOST_ARM64
 
+#if defined(HOST_RISCV64)
+
+enum RiscV64IntrinsicConstants {
+    RiscV64IntrinsicConstants_Zba = 0x0001,
+    RiscV64IntrinsicConstants_Zbb = 0x0002,
+};
+
+#endif // HOST_RISCV64
```

Comment:
```suggestion
#if defined(HOST_RISCV64)
enum RiscV64IntrinsicConstants
{
    RiscV64IntrinsicConstants_Zba = 0x0001,
    RiscV64IntrinsicConstants_Zbb = 0x0002,
};
#endif // HOST_RISCV64
```
Match the formatting of other enums in this file

Reply:
Addressed in https://github.com/dotnet/runtime/pull/113676/commits/6b27297ad6222fb684212b35dbda157ac36bb9ba